sphinx_antsibull_ext/css/antsibull-minimal.scss

```diff
@@ -2,14 +2,16 @@
 
 // Documentation table (for module options and return values)
 table.documentation-table {
-  // Borders and basic spacing
-  border-bottom: 1px solid #000;
-  border-right: 1px solid #000;
+
+  border: 0px;
+
+  th {
+    background-color: #6AB0DE;
```

Comment:
I don't think we should add colors in the antsibull CSS. Colors should be coming from the theme, since these colors will likely not work with different themes.

Reply:
should I set a more subtle color like a `light-grey` here and on the even rows, and override that with the colors in the theme ?

Reply:
That might be a good idea.